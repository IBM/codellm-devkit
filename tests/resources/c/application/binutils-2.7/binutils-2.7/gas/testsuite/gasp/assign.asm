
foo:	.ASSIGNC	"hello"
BAR:	.ASSIGNA	12+34

	\&foo'foo
	\&foo\&foo\&foo 
	\&foo \&foo \&foo 
	\&BAR\&bar\&BAR



	
	.END
