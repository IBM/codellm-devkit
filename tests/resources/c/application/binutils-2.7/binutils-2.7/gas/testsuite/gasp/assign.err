assign.asm:8 Can't find preprocessor variable bar.
