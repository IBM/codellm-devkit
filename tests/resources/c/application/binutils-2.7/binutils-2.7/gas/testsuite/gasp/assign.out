!

!foo:	.ASSIGNC	"hello"
!BAR:	.ASSIGNA	12+34
!

!	\&foo'foo
	hellofoo
!	\&foo\&foo\&foo 
	hellohellohello 
!	\&foo \&foo \&foo 
	hello hello hello 
!	\&BAR\&bar\&BAR
	4646
!

!

!

!	
!	.END
